torch/nn/modules/flatten.py

```diff
@@ -95,14 +95,22 @@ class Unflatten(Module):
 
     __constants__ = ['dim', 'unflattened_size']
     dim: Union[int, str]
-    unflattened_size: Union[Size, NamedShape]
+    unflattened_size: Union[Size, List, NamedShape]
```

Comment:
This doesn't need to be updated because while you're accepting a list as an input to init you're converting that list to a tuple before assigning it to the module's unflattened_size.